test/EntityFramework/FunctionalTests.netcoreapp3.0/FunctionalTests.netcoreapp3.0.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <RootNamespace>System.Data.Entity</RootNamespace>
+    <AssemblyName>EntityFramework.FunctionalTests.netcoreapp3.0</AssemblyName>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="xunit.core" Version="$(XUnitVersion)" />
+    <PackageReference Include="xunit.assert" Version="$(XUnitVersion)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\src\EntityFramework.SqlServer\EntityFramework.SqlServer.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!-- HACK: xUnit.net uses AppDomains on .NET Framework to make App.config work -->
+    <None Update="xunit.console.dll.config">
```

Comment:
☹️ This only makes `build.cmd` work. Test will fail in other test runners (including Visual Studio)

Reply:
What is the underlying issue? Is this related to the fact that a lot of functionality in EF6 still depends on entries in the config file? Assumig that is the case, will the tests work when/if we add sensible default behaviors? Or should the tests change to do code-based configuration?

Reply:
BTW, I am asking about what happens after preview5. I think tis is fine for now.

Reply:
I still need to investigate alternatives. There may be a way to change where ConfigurationManager looks...